smartcar/vehicle.py

```diff
@@ -60,6 +60,28 @@ def permissions(self):
         response = self.api.permissions()
 
         return response.json()['permissions']
+      
+    def has_permissions(self, permissions):
+      """ Checks if vehicle has specified permissions.
+
+        Args:
+            permissions (str or list): the permissions to check on the vehicle
+          
+        Returns:
+            boolean: if vehicle has permissions
+      """
+      vehicle_permissions = self.permissions()
+      prefix = "required:"
+      
+      if isinstance(permissions, list):
+        contained = [permission[permission.startswith(prefix) and len(prefix):] 
```

Comment:
can we break this into two steps, the first that strips the instances of `required:` and the second that checks if it's in `vehicle_permissions`? also I think it might be simpler to use
```
permission.replace('required:', '');